sqlmesh/core/snapshot/evaluator.py

```diff
@@ -625,6 +648,7 @@ def _create_snapshot(
         self,
         snapshot: Snapshot,
         snapshots: t.Dict[SnapshotId, Snapshot],
+        versions_to_create: t.Dict[str, t.List[bool]],
```

Comment:
Feels like we can just pass `t.List[bool]` here, since we only need a record for the provided `snapshot`

Reply:
Also `versions` seems a little overloaded here. Let's be explicit and call it `deployability_flags` or `deployability_flags_to_create`.

Reply:
Yes I was unsure of what to call it to make sense in this context, will alter this back to `deployability_flags`

Reply:
I'm not certain though of how to use the list of the flags for the specific snapshot instead of the whole dict since the function is called in `concurrent_apply_to_snapshots`

Reply:
The function is actually called `_create_snapshot` (singular), no? It just gets passed into `concurrent_apply_to_snapshots` for application over a collection of snapshots

Reply:
Yes my oversight changed this to use the specific snapshot list